lmnet/tests/lmnet_tests/datasets_tests/test_mscoco.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 The Blueoil Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+import pytest
+
+from lmnet.datasets.mscoco import MscocoObjectDetection
+
+
+pytestmark = pytest.mark.usefixtures("set_test_environment")
+
+
+# TODO
+def test_mscoco_segmentation():
```

Comment:
This gives a false sense that we test segmentation (when somebody looks at list of tests)
So just have a todo comment without a function.
`# TODO(takecore): test segmentation`